Marlin/src/pins/gd32f1/pins_SOVOL_V131.h

```diff
@@ -62,6 +62,24 @@
   #define E0_SERIAL_TX_PIN                  PC14
   #define E0_SERIAL_RX_PIN                  PC14
 
+  // Default TMC slave addresses
+  #ifndef X_SLAVE_ADDRESS
+    #define X_SLAVE_ADDRESS                    3
+  #endif
+  #ifndef Y_SLAVE_ADDRESS
+    #define Y_SLAVE_ADDRESS                    3
+  #endif
+  #ifndef Z_SLAVE_ADDRESS
+    #define Z_SLAVE_ADDRESS                    3
+  #endif
+  #ifndef E0_SLAVE_ADDRESS
+    #define E0_SLAVE_ADDRESS                   3
+  #endif
+  static_assert(X_SLAVE_ADDRESS == 3, "X_SLAVE_ADDRESS must be 3 for BOARD_SOVOL_V131.");
+  static_assert(Y_SLAVE_ADDRESS == 3, "Y_SLAVE_ADDRESS must be 3 for BOARD_SOVOL_V131.");
+  static_assert(Z_SLAVE_ADDRESS == 3, "Z_SLAVE_ADDRESS must be 3 for BOARD_SOVOL_V131.");
+  static_assert(E0_SLAVE_ADDRESS == 3, "E0_SLAVE_ADDRESS must be 3 for BOARD_SOVOL_V131.");
```

Comment:
Address 3? That's weird. I'm just making a note to double-check this.

Reply:
I verified it in their config in the SV06 repo:

https://github.com/Sovol3d/Sv06-Source-Code/blob/main/Marlin/Configuration_adv.h

Reply:
See my comment above as well: https://github.com/MarlinFirmware/Marlin/pull/26842#discussion_r1554713169